src/testmenu.cpp

```diff
@@ -0,0 +1,705 @@
+                    TestMenu::_targetAZ = TestMenu::_startAZ + arcmins * AZIMUTH_STEPS_PER_ARC_MINUTE;
+                    mount.moveBy(AZIMUTH_STEPS, arcmins);
+                    _internalState |= DISPLAY_AZ;
+                }
+    #endif
+    #if ALT_STEPPER_TYPE != STEPPER_TYPE_NONE
+                else if (action == F("MoveALTAxis"))
+                {
+                    String output = String(F("Moving ALT axis by ")) + String(_secondaryDistance, 1) + String(F(" arcMins ("));
+                    output += String(ALTITUDE_STEPS_PER_ARC_MINUTE * _secondaryDistance, 0) + " steps) " + actionArg;
+                    Serial.println(output);
+                    float arcmins        = actionArg == "UP" ? _secondaryDistance : -_secondaryDistance;
+                    TestMenu::_startALT  = mount.getCurrentStepperPosition(ALTITUDE_STEPS);
+                    TestMenu::_targetALT = TestMenu::_startALT + arcmins * ALTITUDE_STEPS_PER_ARC_MINUTE;
+                    mount.moveBy(ALTITUDE_STEPS, arcmins);
+                    _internalState |= DISPLAY_ALT;
+                }
+    #endif
+                else if (action == F("ToggleTRK"))
+                {
+                    if (mount.isSlewingTRK())
+                    {
+                        mount.stopSlewing(TRACKING);
+                        Serial.println(F("Tracking stopped."));
+                    }
+                    else
+                    {
+                        mount.startSlewing(TRACKING);
+                        Serial.println(F("Tracking started."));
+                    }
+                    _currentMenu->display();
+                }
+                else if (action == F("FactoryReset"))
+                {
+                    mount.clearConfiguration();
+                    Serial.println(F("Mount reset, EEPROM erased."));
+                    _currentMenu->display();
+                }
+                else if (action = F("PassthroughCmd"))
+                {
+                    Serial.print(F("Enter LX200-OAT command to send to mount: "));
+                    TestMenu::setMenuState(testMenuState_t::WAITING_ON_COMMAND);
+                }
+            }
+            if (TestMenu::getMenuState() == testMenuState_t::CLEAR)
+            {
+                _currentMenu->display();
+            }
+            return;
+        }
+    }
+    Serial.println(F("Invalid key pressed."));
+
+    _currentMenu->display();
+}
+
+void TestMenu::displayStepperPos() const
+{
+    char buffer[64];
+    snprintf(buffer,
+             sizeof(buffer),
+             "  RA: %8ld%s   ALT: %8ld%s   TRK: %8ld%s",
+             mount.getCurrentStepperPosition(RA_STEPS),
+             mount.isAxisRunning(RA_STEPS) ? "^" : " ",
+             mount.getCurrentStepperPosition(ALTITUDE_STEPS),
+             mount.isAxisRunning(ALTITUDE_STEPS) ? "^" : " ",
+             mount.getCurrentStepperPosition(TRACKING),
+             mount.isSlewingTRK() ? "^" : " ");
```

Comment:
The single character strings should be `char`s
```suggestion
             "  RA: %8ld%c   ALT: %8ld%c   TRK: %8ld%c",
             mount.getCurrentStepperPosition(RA_STEPS),
             mount.isAxisRunning(RA_STEPS) ? '^' : ' ',
             mount.getCurrentStepperPosition(ALTITUDE_STEPS),
             mount.isAxisRunning(ALTITUDE_STEPS) ? '^' : ' ',
             mount.getCurrentStepperPosition(TRACKING),
             mount.isSlewingTRK() ? '^' : ' ');
```